pulpcore/cli/common/openapi.py

```diff
@@ -279,11 +348,21 @@ def render_request(
                 else:
                     raise OpenAPIError(_("No suitable content type for request specified."))
         if content_type and validate_body:
-            self.validate_body(
+            body = body or {}
+            if uploads:
+                # Add uploads for validation (CRAZY HACK)
+                body.update(uploads)
+            # This only works, because data or json will be the same object as body...
```

Comment:
Not sure what the second comment means, is it referring to the fact that `validate_schema` returns the same dict object passed in with each key-value pair validated?

As for the hack, not sure what else we can do besides preprocess out the upload fields into a separate schema to check against, but that requires all upload fields to be of the same type. (string: format-binary?)

Reply:
Even more so it processes the objects in the dict in place. Yes.
I think in the long run we may change the outer signature to accept all parts as one body, and separate the files after validation for the request call.